api/src/org/apache/cloudstack/api/command/admin/host/ReconnectHostCmd.java

```diff
@@ -100,17 +103,18 @@ public Long getInstanceId() {
     @Override
     public void execute() {
         try {
-            Host result = _resourceService.reconnectHost(this);
-            if (result != null) {
-                HostResponse response = _responseGenerator.createHostResponse(result);
-                response.setResponseName(getCommandName());
-                this.setResponseObject(response);
-            } else {
-                throw new ServerApiException(ApiErrorCode.INTERNAL_ERROR, "Failed to reconnect host");
-            }
-        } catch (Exception ex) {
-            s_logger.warn("Exception: ", ex);
-            throw new ServerApiException(ApiErrorCode.RESOURCE_UNAVAILABLE_ERROR, ex.getMessage());
+            Host result =_resourceService.reconnectHost(this);
```

Comment:
Before the `result` was checked for `null`; can the method `_resourceService.reconnectHost(this)` return null?

Reply:
nitpick: missing space -> `result = _resourceService`